namespace java com.moseeker.thrift.gen.dao.struct.campaigndb
namespace py thrift_gen.gen.dao.struct.campaigndb


struct CampaignHeadImageDO {

	1: optional i32 id,	//null
	2: optional string imageUrl,	//图片地址
	3: optional string hrefUrl,	//超链接地址
	4: optional string createTime,	//创建时间
	5: optional string updateTime	//创建时间

}
